introduction_to_amazon_algorithms/jumpstart-foundation-models/text2text-fine-tuning-flan-t5.ipynb

```diff
@@ -0,0 +1,596 @@
+{
```

Comment:
<p>Consider adding a description of what's happening next above this cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3916/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
sorry, notebook has changed. can't map the comment anymore